website/docs/r/cwf_process.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: CWF Process"
+sidebar_current: "docs-rightscale-resource-cwf_process"
+description: |-
+  Create and maintain a rightscale CloudWorkFlow process.
+---
+
+# rightscale_cwf_process
+
+Use this resource to create or destroy RightScale [CloudWorkFlow processes](http://docs.rightscale.com/ss/reference/rcl/).
+
+Creating the CWF process will run it synchronously and return the output values (if any).
```

Comment:
Use the present tense: `Creating a CWF process runs it synchronously and returns ...`